### DISCUSSION TITLE
I want my hc hunter back, died to proc of Bonfire Blessing that cannot be turned off

### DISCUSSION THREAD

**Discussion by schtruklyn** [chromiecraft/chromiecraft#7146] [General]
hunter name: Brobro. Was fighting 2 mobs, bear pet was holding aggro, then Blessing proced on my attack, I drew aggro from both mobs, bear couldn't grab aggro, Brobro dead... Blessing cannot be turned off, all in the zone with bonfire have it. This is not hc, because hc ironman toons shouldn't have any buffs. The game is hc unplayable for pet toons with this buff for the next 2 weeks, because pets don't hold aggro after the proc... I want Brobro back!

**Comment by andovcr** [comment]
Hi mate - do you have access to the ChromieCraft discord? You would want to open a #support-ticket there to appeal any hardcore deaths.

**Comment by r-o-b-o-t-o** [comment]
Hello,
Your character has been restored.
In the future, please try to record your gameplay to help with death appeals.
